wrapanapi/systems/container/podman.py

```diff
@@ -0,0 +1,110 @@
+from abc import ABCMeta
+from datetime import datetime
+
+from podman import PodmanClient
+from proto.utils import cached_property
+
+from wrapanapi.entities.base import Entity
+from wrapanapi.systems.base import System
+
+
+class PodmanContainer(Entity, metaclass=ABCMeta):
+    def __init__(self, system, raw=None, **kwargs):
+        """
+        Constructor for an PodmanContainer tied to a specific host.
+
+        Args:
+            system: a Podman system object
+            raw: Raw container object if already obtained, or None
+        """
+        self._name = kwargs.get("name")
+        self._id = kwargs.get("id")
+        if not self._name:
+            raise ValueError("missing required kwargs identifier: 'name'")
```

Comment:
if you're going to raise a ValueError here, you might as well just try a direct access of `name` from the kwargs dict or add it as a positional argument (without default). However, that last section might conflict with existing positional uses.

Reply:
I did not put it as a positional argument because either name or id works for us as a value to that param, so don't want to just say `it has to be name`.

Reply:
But you're checking `if not self._name:` which requires `name` be passed.
If you wanted both, you'd need to check `if not any([self._name, self._id]):`

Reply:
@JacobCallahan Actually, name or id is an identifier, and any of them was required. So instead of having two different params now, I have changed the implementation to pass positional arg now named `key`. User can still pass name and id as optional arg though.

Key param accepts either name or id. Same param name is being used by podman python module